src/content/glossary/index.md

```diff
@@ -681,6 +681,14 @@ A program that executes on the Ethereum computing infrastructure.
   Introduction to Smart Contracts
 </DocLink>
 
+### SNARK {#snark}
+
+Succinct Non-interactive ARgument of Knowledge a type of a [zero-knowledge proof](#zk-proof).
+
+<DocLink to="/developers/docs/scaling/layer-2-rollups/#zk-rollups">
+  Zero-knowledge Rollups
```

Comment:
Unsure if we want this link in every kind-of related occurrence here. Anyone have any thoughts?

```suggestion
  Zero-knowledge rollups
```